lib/plug/conn/wrapper_error.ex

```diff
@@ -20,6 +20,8 @@ defmodule Plug.Conn.WrapperError do
 
   @deprecated "Use reraise/1 or reraise/4 instead"
   def reraise(conn, kind, reason) do
+    # Please do not submit a PR to remove System.stacktrace()
+    # See https://github.com/elixir-plug/plug/pull/991
```

Comment:
Small suggestion: let's also add a TODO for us to remove this when we depend on Elixir 1.10+:

```suggestion
    # Please do not submit a PR to remove System.stacktrace()
    # See https://github.com/elixir-plug/plug/pull/991
    # TODO: use __STACKTRACE__ instead of System.stacktrace/0 when we depend on Elixir 1.10+.
```